### ISSUE TITLE
Absolute error given by `scipy.integrate.quad`

### ISSUE THREAD

**Issue by wellington36** [wellington36/tweedie-numerical-approximation#5]
Is the absolute error given by `scipy.integrate.quad` a good estimate? Se this results in the naive version of Tweedie's pdf distribution:

|alpha | points | value                   | absolute error (estimation)           |
|---------|-----------|-----------------|-----------------|
|0.01   | 1000     |$\infty$| $\infty$|
|0.1     | 1000     |1.0000e+00| 1.0265e-05|
|0.2     | 1000     |1.0000e+00| 1.4301e-06|
|0.3     | 1000     |1.0000e+00| 1.6821e-04|
|0.4     | 1000     |0.9670e+00| 6.5222e-02|
|0.5     | 1000     |1.0085e+00| 1.8860e-01|
|0.6     | 1000     |$-\infty$| $\infty$|
|0.7     | 1000     |0.0000e+00| 0.0000e-00|
|0.8     | 1000     |0.0000e+00| 0.0000e-00|
|0.9     | 1000     |0.0000e+00| 0.0000e-00|
|0.99   | 1000     |0.0000e+00| 0.0000e-00|

**Comment by wellington36** [comment]
The answer to the above question is generally yes. But in our case, maybe not so interesting. Why the objective is different, first let's see what is the absolute error estimate given by the `quad`.

We don't have many references about this estimate, just the FORTRAN code ([here](https://github.com/scipy/scipy/blob/cc9f6b203742db6e5ec30a6a15f4a2c292e21ffb/scipy/integrate/quadpack/dqk21.f)). The point here is as we don't have the actual result of the integral to compare we use a mixture of other methods calculated on a fixed number of points (a better explanation can be found [here](https://stackoverflow.com/questions/41662492/what-is-the-meaning-of-error-obtained-in-dblquad-integration-using-scipy)), in general this form works well as long as you don't have a high numerical error. In our case we have the opposite, we know the result of the integration of a pdf that is 1 what we want to check is the errors in the sum of the terms.

In conclusion, more explanatory data can be found in [[1](#dias)] calculating $|1 - result|$.

[1]<a id="dias"></a>: DIAS NL, RIBEIRO JR PJ. Practical rules for summing the series of the Tweedie probability density function with high-precision arithmetic. An Acad Bras Ciênc [Internet]. 2019;91(4):e20180268. Available from: https://doi.org/10.1590/0001-3765201920180268

**Comment by wellington36** [comment]
Closed by 6cea2178ab5c37b3c076a77ddceb0246f7d73784
